tests/core/failover/task.yaml

```diff
@@ -21,30 +21,58 @@ environment:
     INJECT_FAILURE/emptysystemd: inject_emptysystemd_failure
     # FIXME: disabled until we find what to do!
     # fails with:  ERROR cannot replace signed kernel snap with an unasserted one
-    #INJECT_FAILURE/emptyinitrd: inject_emptyinitrd_failure
+    INJECT_FAILURE/emptyinitrd: inject_emptyinitrd_failure
     TARGET_SNAP/rclocalcrash: core
     TARGET_SNAP/emptysystemd: core
-    #TARGET_SNAP/emptyinitrd: kernel
+    TARGET_SNAP/emptyinitrd: kernel
     BUILD_DIR: /home/tmp
     # uploading the core or otherwise large snap triggers OOM
     SNAPD_NO_MEMORY_LIMIT: 1
+    BLOB_DIR: $(pwd)/fake-store-blobdir
+
+    SNAP_ID_core20: DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q
+    SNAP_ID_core18: CSO04Jhav2yK0uz97cr0ipQRyqg0qQL6
+    SNAP_ID_core: 99T7MUlRhtI3U0QFgl5mXXESAiSwt776
+    SNAP_ID_pc_kernel: pYVQrBcKmBa0mZ4CCN7ExT6jH8rY1hza
```

Comment:
We will have some problems on arm devices, we can either extend this to also pi_kernel or just run the kernel variant on i386 and amd64.

Reply:
I've limited the kernel variant to run on non-arm architectures only